### ISSUE TITLE
Full precision accuracy for ResNet-20 pre-trained with CIFAR-10 in the paper appears to be inconsistent

### ISSUE THREAD

**Issue by OedoSoldier** [lihuantong/HAST#2]
In your paper, the ResNet-20 model pre-trained with CIFAR-10 achieves a full precision accuracy of 94.03% (as shown in the image below):

![](https://i.imgur.com/5RfKqCk.png)

However, in other papers that focus on zero-shot quantization or data-free quantization (such as the ones linked below: https://arxiv.org/abs/2303.06869, https://arxiv.org/abs/2111.02625), the reported full-precision accuracy for the same pre-trained model is 93.89% (as shown in the image below):

![](https://i.imgur.com/39TBcz3.png)

This discrepancy raises the question of why these accuracies are inconsistent.

**Comment by OedoSoldier** [comment]
Per I test, the accuracy for this model in `pytorchcv` is indeed 93.89%

![](https://i.imgur.com/RUrdABJ.png)

**Comment by lihuantong** [comment]
Model in different versions of the pytorchcv library has different accuracy. You can see in Readme that the pytorch version used in the paper is 1.8.1, which is consistent with the benchmark [https://arxiv.org/abs/2111.09136](https://arxiv.org/abs/2111.09136).
